elastic/kibana
# [Security Solution][Alert details] - fix z-index of response details flyout to be shown above timeline

## Summary

This PR fixes an issue introduced in [this PR](https://github.com/elastic/kibana/pull/177087) where - in order to get the expandable flyout to work within timeline - we had to modify the `z-index` of the timeline modal ([here](https://github.com/elastic/kibana/pull/177087/files#diff-a4fef8db9f9e4c1a86b8fcc0f859b10420a1ff0c1c5c091eace92cf0643ca58c)) as well as the _secondary flyouts_ accessed through the `Take action` button in the flyout's footer.

I had missed this flyout opened from the Reponse Details tab.

#### Before fix

https://github.com/elastic/kibana/assets/17276605/0e73b93a-f6c9-4952-8330-85d521d4e209

#### After fix

https://github.com/elastic/kibana/assets/17276605/f5b5fd1b-d679-458d-8d31-f70504bc8dd9

### How to test

Make sure you have the `expandableTimelineFlyoutEnabled` feature flag in your `kibana.yml` config file.

- either generate data for the response tab to be populated

- or force the `QueryDetailsFlyout` to always be shown by modifying these few lines [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/osquery/public/live_queries/form/pack_queries_status_table.tsx#L423-L425). That was as soon as you open the left panel of the expandable flyout, the query details flyout will be rendered. You'll also need to hardcoded some data [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/common/components/event_details/osquery_tab.tsx#L81) for the view to show some data

https://github.com/elastic/kibana/issues/181235
